cmd/list_test.go

```diff
@@ -74,12 +75,6 @@ func TestListV2(t *testing.T) {
 		t.Error("list command should contain id 'java-microprofile'")
 	}
 
-	output, _ = cmdtest.RunAppsodyCmdExec([]string{"list", "appsodyhub"}, ".")
-
-	// we expect 2 instances
-	if !(strings.Count(output, "java-microprofile") == 1) {
```

Comment:
I had some issues with this test as well, and I think it needs to be reworked. However this part is not failing for me, I was getting issues with the `strings.Count(output, "nodejs ") == 2` a couple lines down. What was the reason for removing this?

Reply:
I had also errors with the `"nodejs "` line, but it gets fixed when you run the whole test again because the previous test adds a new repository with a `nodejs` stack.

I removed this because the same check is performed when doing the following, just above.
```go
if !strings.Contains(output, "java-microprofile") {
	t.Error("list command should contain id 'java-microprofile'")
}
```